docs/monthly/2023-11-monthly.md

```diff
@@ -0,0 +1,165 @@
+
+- In general we prepared prepared multiple of these 💭 idea items and many of them will turn into 💬 features soon.
+
+## Hydra development
+
+[Issues and pull requests closed since the last
+report](https://github.com/input-output-hk/hydra/issues?q=is%3Aclosed+sort%3Aupdated-desc+closed%3A2023-10-31..2023-11-30)
+
+This month, the team worked on the following:
+
+### MeshJS + Hydra
+
+At the beginning of month we attended the Cardano Summit in Dubai where we held
+the _Hydra MasterClass_ workshop. There we also presented the Hydra-Poll dApp
+which is using MeshJS to build Hydra transactions on the frontend.
+
+We collaborated with the MeshJS team closely and as the outcome of this work
+they added some Hydra specific features to the MeshJS SDK. There is a nice low
+level [API](https://meshjs.dev/apis/transaction/builderExample) for building
+custom transactions which is pretty similar to what `cardano-cli` does.
+
+Now you can use `isHydra` field which you can pass to this API which then uses
+another set of protocol paramters suitable for Hydra (eg. usually in Hydra we
+want to have zeroed transaction fees, reduce script execution units etc.).
+
+Big thanks to the MeshJS team on giving us much needed support! They were
+presenting these changes in our monthy review meeting so make sure to watch the
+[recording](https://drive.google.com/file/d/1-iv8IveUzA2KrJV_Kqrgx4ts05Ow0zjM).
+
+### Rendering the Hydra specification
+
+Besides working on the design of incremental commits itself, we also wanted to
+tidy up the way the [Hydra protocol
+specification](https://hydra.family/head-protocol/core-concepts/specification)
+is written itself.
+
+So far, the specification was written in LaTex and rendered to a PDF. LaTeX was
+a good choice because of its expressiveness on math expressions, but not so
+great of a choice for pull requesting changes and the entry barrier is quite
+high. The majority of documentation around Hydra and Mithril is written using
+Markdown and rendered into a webpage using a static site generator, currently
+[Docusaurus](https://docusaurus.io/). Most modern markdown renderers do support
+LaTeX-style math nowadays, even the normal [Github file
+preview](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions).
+
+Hence, we experimented with using the more standard Markdown format to write the
+specification, but still being able to use LaTeX math **and** LaTeX macros
+(which are _still_ heavily used in the spec). From this we want to have both, a
+PDF and HTML rendering.
+
+![](https://ipfs.io/ipfs/QmPUTYaSViLEyZcGmsTPL4jHdGSevH3yc3RD7TRhU3ivwH?filename=Peek%202023-11-28%2018-42.gif)
+
+While not yet finished, the GIF above shows already progress being made using
+[pandoc](https://pandoc.org/) to resolve macros, producing a PDF via `xelatex`
+and Docusaurus compatible markdown. Using powerful extensions of pandoc we can
```

Comment:
```suggestion
and Docusaurus compatible markdown. Using powerful extensions of pandoc one can
```

Reply:
Please also update 'markdown' to 'Markdown'